### PR TITLE
chore(99-general-concepts): agrega lectura sobre cómo depurar

### PR DESCRIPTION
**Pull request by ivandevp**
@nicolethenerd this is the second reading about how to debug JS code

### CODE REVIEW

File: 99-general-concepts/01-debugging/03-breakpoints/README.md [hunk 1]
```diff
@@ -1,16 +1,159 @@
...
+![Chrome Dev Tools: Sources](https://cdn-images-1.medium.com/max/800/1*JhVFiJNoGZAAmlnk00y6TA.gif)
+
+Para la depuración del código JS, podemos hacer uso de **breakpoints**, que son
+puntos donde queremos que nuestro código se detenga en tiempo de ejecución.
+
+En el siguiente ejemplo, tenemos un código que verifica si el número ingresado
+por un usuario es cero, positivo o negativo.
+
+```javascript
+var number = prompt("Ingrese un número");
```
Comment (nicolethenerd) [inline 99-general-concepts/01-debugging/03-breakpoints/README.md]: single quotes, according to our new style guide

Comment (ivandevp) [reply 99-general-concepts/01-debugging/03-breakpoints/README.md]: You're right! I'll fix it right now 👍

Comment (ivandevp) [reply 99-general-concepts/01-debugging/03-breakpoints/README.md]: Now it's fixed 😃

File: 99-general-concepts/01-debugging/03-breakpoints/README.md [hunk 2]
```diff
@@ -1,16 +1,161 @@
...
+Para poner en práctica esta lectura, puedes depurar el siguiente código:
+
+```javascript
+/*
+ * Corrige el siguiente código de tal forma que obtengas el resultado esperado,
+ * usa las técnicas de depuración que conoces para encontrar y solucionar
+ * el error.
+ */
+
+var reverse = function (text) {
```
Comment (nicolethenerd) [inline 99-general-concepts/01-debugging/03-breakpoints/README.md]: no space, according to the style guide

**Pull request merged**